src/AppBundle/Resources/views/Ndr/Asset/Property/_list_item.html.twig

```diff
@@ -14,138 +14,241 @@
-            </td>
-        </tr>
-
-        {# other charges  #}
-        <tr>
-            <td>{{ 'form.property.hasCharges.label' | trans(transOptions) }}</td>
-            <td>{{ asset.hasCharges | capitalize }}</td>
-            <td class="change-answer soft-half--left hard--right width-sixth">
-                <a class="behat-link-edit action-link"
-                   href="{{ path('ndr_assets_property_step', {'ndrId': ndr.id, 'assetId': asset.id, 'step': 7}) }}">{{ 'edit' | trans({}, 'common' ) }}</a>
-            </td>
-        </tr>
+<dl class="govuk-summary-list behat-region-list-properties">
+DELETE ME
+    <div class="govuk-summary-list__row">
+        <dt class="govuk-summary-list__key">
+            {{ 'question' | trans({}, 'common') }}
+        </dt>
+        <dd class="govuk-summary-list__value">
+            {{ 'answer' | trans({}, 'common') }}
+        </dd>
+        <dd class="govuk-summary-list__actions">
+            <a class="govuk-link" href="#">
+                <span class="visually-hidden">{{ 'actions' | trans({}, 'common') }}</span>
+            </a>
+        </dd>
+    </div>
 
-        {# rented out + rent agreement date + income month #}
-        <tr>
-            <td>{{ 'form.property.isRentedOut.label' | trans(transOptions) }}</td>
-            <td>{{ asset.isRentedOut | capitalize }}</td>
-            <td class="change-answer soft-half--left hard--right width-sixth">
-                <a class="behat-link-edit action-link"
-                   href="{{ path('ndr_assets_property_step', {'ndrId': ndr.id, 'assetId': asset.id, 'step': 8}) }}">{{ 'edit' | trans({}, 'common' ) }}</a>
-            </td>
-        </tr>
-        {% if asset.isRentedOut =='yes' %}
-            <tr>
-                <td>{{ 'form.property.rentAgreementEndDate.label' | trans(transOptions) }}</td>
-                <td>{{ asset.rentAgreementEndDate | date("m/Y") }}</td>
-                <td class="change-answer soft-half--left hard--right width-sixth">
-                    <a class="behat-link-edit action-link"
-                       href="{{ path('ndr_assets_property_step', {'ndrId': ndr.id, 'assetId': asset.id, 'step': 8}) }}">{{ 'edit' | trans({}, 'common' ) }}</a>
-                </td>
-            </tr>
-            <tr>
-                <td>{{ 'form.property.rentIncomeMonth.label' | trans(transOptions) }}</td>
-                <td>£{{ asset.rentIncomeMonth | money_format }}</td>
-                <td class="change-answer soft-half--left hard--right width-sixth">
-                    <a class="behat-link-edit action-link"
-                       href="{{ path('ndr_assets_property_step', {'ndrId': ndr.id, 'assetId': asset.id, 'step': 8}) }}">{{ 'edit' | trans({}, 'common' ) }}</a>
-                </td>
-            </tr>
-        {% endif %}
+  
+    <div class="govuk-summary-list__row">
+        <dt class="govuk-summary-list__value">
```

Comment:
This should be `govuk-summary-list__key`. There may be others like this (should be easy to search for `<dt />`s with this class)